help/destinations/catalog/personalization/pega-profile.md

```diff
@@ -0,0 +1,134 @@
+>id="platform_destinations_connect_s3_bucket"
+>title="Bucket name"
+>abstract="Must be between 3 and 63 characters long. Must begin and end with a letter or number. Must contain only lowercase letters, numbers, or hyphens ( - ). Must not be formatted as an IP address (for example, 192.100.1.1)."
+
+>[!CONTEXTUALHELP]
+>id="platform_destinations_connect_s3_folderpath"
+>title="Folder path"
+>abstract="Must contain only characters A-Z, a-z, 0-9 and can include the following special characters: `/!-_.'()"^[]+$%.*"`. To create a folder per segment file, insert the macro `/%SEGMENT_NAME%` or `/%SEGMENT_ID%` or `/%SEGMENT_NAME%/%SEGMENT_ID%` into the text field. Macros can only be inserted at the end of the folder path. View macro examples in the documentation."
+>additional-url="https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/cloud-storage/overview.html#use-macros" text="Use macros to create a folder in your storage location"
+
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**. An asterisk next to a field in the UI indicates that the field is required.
+
+* **[!UICONTROL Name]**: enter a name that will help you identify this destination.
+* **[!UICONTROL Description]**: enter a description of this destination.
+* **[!UICONTROL Bucket name]**: enter the name of the [!DNL Amazon S3] bucket to be used by this destination.
+* **[!UICONTROL Folder path]**: enter the path to the destination folder that will host the exported files.
+* **[!UICONTROL Compression Type]**: Select compression type as GZIP or NONE.
+
+>[!TIP]
+>
+>In the connect destination workflow, you can create a custom folder in your Amazon S3 storage per exported segment file. Read [Use macros to create a folder in your storage location](overview.md#use-macros) for instructions.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+See [Activate audience data to batch profile export destinations](../../ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.
+
```

Comment:
there is a subsection which would come in handy here, about the mapping step. see the self-service documentation template: 

https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#map

Could we perhaps add this section and briefly describe the particularities of the mapping step for this new Pega destination?

Reply:
Mapping need to be done in Pega Customer Decision Hub side. I tried adding few predefined fields in AEP side, however, the connector complains that the source array field cannot be mapped to String. Also, if I use predefined Attributes, I was not able to create custom target field. so revert back my changes

Reply:
@paratAtPega - Apologies, I missed your reply (including this comment). Perhaps you can reach out to our partnership team to inquire about the error you are seeing? 

Also, just wondering, since this is a file export integration, I would think that users will export files that include attributes and maybe also identities. Is that correct? I think it would be valuable to show that in a mapping step in the documentation, similar to what other destinations have done (see examples in https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#map).

Also, apart from this comment, the documentation looks great. We'll publish this documentation page after the Pega team submits the publish request for the destination and our team approves it.

Reply:
Added mapping paragraph after referencing other file based connectors. (google-cloud)